Pomodoro.DataAccess/Entities/Completed.cs

```diff
@@ -9,7 +9,7 @@ public class Completed : BaseEntity
         public Guid TaskId { get; set; }
         public DateTime ActualDate { get; set; }
         public int TimeSpent { get; set; }
-        public float PomodorosCount { get; set; }
+        public int PomodorosCount { get; set; }
```

Comment:
In my opinion, to have more accurate statistics, the type should be float. Because `TimeSpent` won't always be dividable by the pomodoro duration without a remainder. But in statistics this value should be rounded to the nearest integer.